concordium-consensus/src/Concordium/KonsensusV1/Consensus.hs

```diff
@@ -19,9 +19,12 @@ import Concordium.Types.BakerIdentity
 
 -- |A Monad for multicasting timeout messages.
 class MonadMulticast m where
```

Comment:
This should probably be generalized over any type `a` that is an instance of `Serialize`.